assets/src/types/react-swm-icon-pack/index.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module "react-swm-icon-pack" {
+  import { FC } from "react";
+
+  type IconProps = {
+    color: string;
+    strokeWidth: string | number;
+    size: string | number;
+    set: "broken" | "curved" | "duotone" | "outline";
+  };
+
+  export type Icon = FC<Partial<IconProps> & React.HTMLAttributes<HTMLDivElement>>;
+}
```

Comment:
Would it be possible to upstream this?

Reply:
The swm icon pack is owned by a person not working in SWM so it would be necessary to fork the repo, make a PR there and wait for the changes (which pessimistically can take a long time). In my opinion this looks good and serves the needs of the project.

Reply:
Well, patching a dependency locally doesn't look good, especially if the same has to be done in two projects. There's always a risk of these types getting out of sync with the main package. For me, the way to go is to fork the package and use it as git dependency (npm seems to handle that, although I have no experience with such an approach)